docs/develop/GSOC/integrated_plot_gallery.md

```diff
@@ -15,10 +15,12 @@ exclude: true
   Besides existing parameters new have to become enabled, e.g. mapsection, projection. 
   An extension to a larger project is to add an interactive user interface.
 
-* **Duration**: 175h - 350h
-* **Skills**: Python, QT, UI programming, Git
+* **Duration**: 90h - 350h
+* **Skills**: Python, QT, UI programming, Git, AI-Assistant
 * **Difficulty level**: Medium
 * **Related Readings/Links**:
   * https://mss.readthedocs.io/en/stable/gallery/index.html
   * https://mss.readthedocs.io/en/stable/mswms.html?highlight=gallery#standalone-server-setup
+  * https://mss.readthedocs.io/en/stable/tutorial.html
```

Comment:
```suggestion
  * <https://mss.readthedocs.io/en/stable/tutorial.html>
```
Adding angle brackets should make these links actually click-able on the website (since that is markdowns link syntax for links that should not have a different text).